Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: help install run start test lint format
+
+help: ## Print all commands (default)
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+####### BUILD #######
+
+install: ## Install dependencies
+	npm run build
+
+####### RUN #######
+
+run: ## Run the 15-puzzle isomorphic app
+	npm run start
```

Comment:
My other project app also runs on 3000, can you let me specify port through an environment variable ?
Something like `PORT=3001 make run` would be great !